bin/generate_missing_mappings

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative "../config/environment"
+require "qdrant"
+require "bigdecimal"
+require "openai"
+require "csv"
+require "yaml"
+
+class TaxonomyMapper
+  MAX_RETRIES = 3
+  QDRANT_PORT = 6333
+  EMBEDDING_MODEL = "text-embedding-3-small"
+  MAPPING_GRADER_GPT_MODEL = "gpt-4"
+  LATEST_SHOPIFY_VERSION = "shopify/#{File.read("VERSION").strip}"
+
+  def initialize
+    @openai_client = create_openai_client
+    @qdrant_client = create_qdrant_client
+  end
+
+  def run
+    shopify_categories_missing_mapping_groups = find_unmapped_shopify_categories
+    return if shopify_categories_missing_mapping_groups.empty?
+
+    shopify_categories_missing_mapping_groups.each do |missing_mapping_group|
+      generate_missing_mappings(missing_mapping_group)
+    end
+  end
+
+  private
+
+  def create_openai_client
+    OpenAI::Client.new(
+      access_token: ENV["OPENAI_API_TOKEN"],
+      uri_base: "https://openai-proxy.shopify.ai/v1",
+      request_timeout: 10,
+    )
+  end
+
+  def create_qdrant_client
+    ensure_qdrant_server_running
+    Qdrant::Client.new(url: "http://localhost:#{QDRANT_PORT}")
+  end
+
+  def ensure_qdrant_server_running
+    return if system("lsof -i:#{QDRANT_PORT}", out: "/dev/null")
+
+    command = "podman run -p #{QDRANT_PORT}:#{QDRANT_PORT} qdrant/qdrant"
```

Comment:
I don't think `podman` is a common out-of-box command. We need to at least include a note about this in the README

Reply:
Added a command `make setup-mapping-generation-tool` to install and configure mapping generation tool.
Also updated README and `make help` to include its description.